.jenkins/pytorch/macos-test.sh

```diff
@@ -15,7 +15,7 @@ if [ ! -d "${PYTORCH_ENV_DIR}/miniconda3" ]; then
 fi
 export PATH="${PYTORCH_ENV_DIR}/miniconda3/bin:$PATH"
 source ${PYTORCH_ENV_DIR}/miniconda3/bin/activate
-conda install -y mkl mkl-include numpy pyyaml setuptools cmake cffi ninja future six
+conda install -y mkl mkl-include numpy pyyaml setuptools cmake cffi ninja six
```

Comment:
`future` doesn't have Python 3.7 support, but it seems that we don't actually need it.